### PR TITLE
feat: Show Primal Core in one line + Tooltips

### PR DESCRIPTION
**Pull request by Fia3D**
image
image

### CODE REVIEW

File: SavedInstances/Core/Core.lua [hunk 1]
```diff
@@ -2514,6 +2514,44 @@ hoverTooltip.ShowDragonflightRenownTooltip = function (cell, arg, ...)
...
+  local toonstr = (db.Tooltip.ShowServer and toon) or strsplit(' ', toon)
+
+  indicatortip:AddHeader(ClassColorise(t.Class, toonstr), string.format("%d/%d", totalDone, #P.TrackedQuest[index].relatedQuest))
+
+  local stringTypeCore = {
+    YELLOW_FONT_COLOR_CODE .. "Earth" .. FONT_COLOR_CODE_CLOSE,
+    "|cff42a4f5" .. "Water" .. FONT_COLOR_CODE_CLOSE,
+    "|cffe4f2f5" .. "Air" .. FONT_COLOR_CODE_CLOSE,
+    ORANGE_FONT_COLOR_CODE .. "Fire" .. FONT_COLOR_CODE_CLOSE
+  }
```
Comment (LiangYuxuan) [inline SavedInstances/Core/Core.lua:2542]: Use localize string like `L["Earth"]` to allow localization.

File: SavedInstances/Core/Core.lua [hunk 2]
```diff
@@ -2514,6 +2514,44 @@ hoverTooltip.ShowDragonflightRenownTooltip = function (cell, arg, ...)
...
+  local IDs = P.TrackedQuest[index].relatedQuest
+
+  for i, questID in ipairs(IDs) do
+    indicatortip:AddLine(
+      stringTypeCore[i],
+      t.Progress[index][questID] and GREENFONT .. L["Obtained"] .. FONTEND or "Unobtained"
```
Comment (LiangYuxuan) [inline SavedInstances/Core/Core.lua:2548]: Use global string. https://github.com/SavedInstances/SavedInstances/blob/92142fcd6a380cfcaa8c1019e042578afed8983d/SavedInstances/Core/Core.lua#L2412-L2413

File: SavedInstances/Modules/Progress.lua [hunk 3]
```diff
@@ -310,6 +310,44 @@ local function DragonflightRenownReset(toon, index)
...
+  for i, questID in ipairs(Module.TrackedQuest[index].relatedQuest) do
+    if t.Progress[index][questID] then
+      if finalString ~= "" then
+        finalString = finalString .. ' / '
+      end
+      finalString = finalString .. stringTypeCore[i]
+    end
+  end
+  return finalString == "" and "None" or finalString
+end
```
Comment (LiangYuxuan) [inline SavedInstances/Modules/Progress.lua:342]: I'd prefer use number progress like `1/4` as first letter abbr is hard to localize and understand in other languages.

**Review by LiangYuxuan — CHANGES_REQUESTED** [review]
Thank you! Mainly on localization issues to solve.

**Comment by Fia3D** [comment]
> Thank you! Mainly on localization issues to solve.

I have fixed everything you have pointed, thank you

**Review by LiangYuxuan — DISMISSED** [review]

**Comment by Stanzilla** [comment]
Looks good but needs a rebase now!

**Comment by Fia3D** [comment]
> Looks good but needs a rebase now!

Done

**Review by Stanzilla — APPROVED** [review]

**Pull request merged**